shell/common/shell.cc

```diff
@@ -427,7 +427,7 @@ Shell::~Shell() {
 
   fml::TaskRunner::RunNowOrPostTask(
       task_runners_.GetUITaskRunner(),
-      fml::MakeCopyable([engine = std::move(engine_), &ui_latch]() mutable {
+      fml::MakeCopyable([&engine = engine_, &ui_latch]() mutable {
```

Comment:
I think it's more straightforward to capture `this` and then use `engine_.reset()`